Softeq.XToolkit.WhiteLabel/Bootstrapper/Abstract/IContainerBuilder.cs

```diff
@@ -29,6 +29,8 @@ public interface IContainerBuilder
 
         void Singleton<TService>(Func<IContainer, TService> func, IfRegistered ifRegistered = IfRegistered.AppendNew);
 
+        void SingletonType<TType>(IfRegistered ifRegistered = IfRegistered.AppendNew);
```

Comment:
The method name doesn't describe it enough to use and override (could you please rename and add documentation?)

Reply:
Autofac also has extension to register instance as all of implemented interfaces: https://autofac.org/apidoc/html/5BE7EA96.htm Which other DI implementation do we need to consider to be potentially supported?
Also, regarding the custom `IContainerBuilder` interface, - in this case we would have to also implement brand new `Bootstrapper` hierarchy, or extend `Bootstrapper` to support any container implementation